### PR TITLE
Added brams.md

### PR DESCRIPTION
**Pull request by Voninkazo**
Hey, 
Could please check my PR and suggest any changes, I looked for the information, but there wasn't.

Thanks
Sandy

### CODE REVIEW

File: b/brams.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Brams
+
+![Brams](brams.jpg)
+
+Unfortunately, we could not find any information about this singer. We only found his homepage, so you can have a lookat that.
```
Comment (ndorfin) [inline b/brams.md]: `look at`

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Comment by Voninkazo** [comment]
I've made the change,
Could you please check again.

**Review by ndorfin — APPROVED** [review]
Looks good. Ship-it!

**Comment by Voninkazo** [comment]
Thank you.

**Pull request merged**